app/src/main/res/layout/activity_view_history.xml

```diff
@@ -26,7 +26,7 @@
         app:layout_constraintStart_toStartOf="parent" />
 
     <TextView
-        android:id="@+id/Label"
+        android:id="@+id/logLbl"
```

Comment:
I would change this to something other than Label... but probably something more descriptive than logLbl on the same note.